src/Catalyst.Core.Modules.Consensus.Tests/UnitTests/Deltas/DeltaBuilderTests.cs

```diff
@@ -1,341 +1,343 @@
-                _cache, _dateTimeProvider, _logger);
-            var candidate = deltaBuilder.BuildCandidateDelta(_previousDeltaHash);
-
-            ValidateDeltaCandidate(candidate, expectedBytesToHash);
-
-            _cache.Received(1).AddLocalDelta(Arg.Is(candidate), Arg.Any<Delta>());
-        }
-
-        private static TransactionBroadcast[] BuildSelectedTransactions(List<TransactionBroadcast> transactions)
-        {
-            var selectedTransactions = transactions.Where(t => (t.IsPublicTransaction || t.IsContractCall || t.IsContractDeployment) && t.HasValidEntries()).ToArray();
-            return selectedTransactions;
-        }
-
-        private byte[] BuildExpectedBytesToHash(TransactionBroadcast[] selectedTransactions, byte[] shuffledEntriesBytes)
-        {
-            var signaturesInOrder = selectedTransactions
-               .Select(p => p.Signature.ToByteArray())
-               .OrderBy(s => s, ByteUtil.ByteListComparer.Default)
-               .SelectMany(b => b)
-               .ToArray();
-
-            var expectedCoinBase = new CoinbaseEntry
-            {
-                Amount = selectedTransactions.Sum(t => t.SummedEntryFees()).ToUint256ByteString(),
-                ReceiverPublicKey = _producerId.PublicKey.ToByteString()
-            };
-
-            var expectedBytesToHash = shuffledEntriesBytes.Concat(signaturesInOrder)
-               .Concat(expectedCoinBase.ToByteArray()).ToArray();
-            return expectedBytesToHash;
-        }
-
-        private void ValidateDeltaCandidate(CandidateDeltaBroadcast candidate, byte[] expectedBytesToHash)
-        {
-            candidate.Should().NotBeNull();
-            candidate.ProducerId.Should().Be(_producerId);
-            candidate.PreviousDeltaDfsHash.ToByteArray().SequenceEqual(_previousDeltaHash.ToArray()).Should().BeTrue();
-
-            var expectedHash = CidHelper.CreateCid(_hashProvider.ComputeMultiHash(expectedBytesToHash));
-            candidate.Hash.ToByteArray().Should().BeEquivalentTo(expectedHash.ToArray());
-        }
-    }
-}
+//#region LICENSE
+
+///**
+//* Copyright (c) 2019 Catalyst Network
+//*
+//* This file is part of Catalyst.Node <https://github.com/catalyst-network/Catalyst.Node>
+//*
+//* Catalyst.Node is free software: you can redistribute it and/or modify
+//* it under the terms of the GNU General Public License as published by
+//* the Free Software Foundation, either version 2 of the License, or
+//* (at your option) any later version.
+//*
+//* Catalyst.Node is distributed in the hope that it will be useful,
+//* but WITHOUT ANY WARRANTY; without even the implied warranty of
+//* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+//* GNU General Public License for more details.
+//*
+//* You should have received a copy of the GNU General Public License
+//* along with Catalyst.Node. If not, see <https://www.gnu.org/licenses/>.
+//*/
+
+//#endregion
+
+//using System;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/app/catalyst-network/Catalyst.Framework/pullRequest?prid=4562326)